lxd/images.go

```diff
@@ -4010,43 +4043,82 @@ func imageExport(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(err)
 	}
 
-	// Get the image (expand the fingerprint).
+	isDevLXDQuery := r.RemoteAddr == "@devlxd"
```

Comment:
Looks like we could do with a constant for `@devlxd`

Reply:
Done.